BuildGuide.md

```diff
@@ -0,0 +1,126 @@
+# Build Guide
+
+***Disclaimer***
+
+While this guide is meant to give Iron180 owners a comprehensive overview of how to build a 180, please keep in mind that there are many variations/alternatives to some of the processes described here. At the end of the day, some of these choices come to preference, so do not worry if you see other individuals building boards in a different way. That being said, if you are more of a visual person and would rather see videos, we have linked a few links to build streams below.
```

Comment:
nits: 
(1) consider `assemble` instead of `build`
(2) `comes down to preference` instead of `come to preference`
(3) `provided a few links` instead of `linked a few links`

Reply:
integrated 2 and 3, left one since we are calling it a build guide.